package.json

```
@@ -21,7 +21,9 @@
     "changeset": "changeset",
     "release": "changeset publish",
     "postinstall": "husky install",
-    "format": "prettier -c --write \"*/**\""
+    "format": "prettier -c --write \"*/**\"",
+    "test": "jest",
+    "test:watch": "jest --watch"
```

It'd be useful to print out test coverage info in the CI terminal

```suggestion
    "test": "jest",
    "test:watch": "jest --watch",
    "test:ci": "jest --env=jsdom --coverage --watchAll=false --maxWorkers=2",
```